### PR TITLE
Tests for Nightly Build benchmark

### PR DESCRIPTION
**Pull request by bodhinandach**
the title says. Can we approve this quickly? I will do a follow up PR for the documentation.

### CODE REVIEW

File: explicit_single_phase/validation/sliding_block_inclined_boundary/mpm-file.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      "resume": false,
+      "uuid": "boundary_inclined_friction",
+      "step": 0
+    }
+  },
+  "post_processing": {
+    "path": "results/",
+    "output_steps": 1000
+  }
+}
```
Comment (jgiven100) [inline explicit_single_phase/validation/sliding_block_inclined_boundary/mpm-file.json:66]: whats the difference between `mpm.json` and `mpm-file.json`?

Comment (jgiven100) [reply explicit_single_phase/validation/sliding_block_inclined_boundary/mpm-file.json:66]: in other words, do we need both?

File: explicit_single_phase/validation/sliding_block_inclined_boundary/test_benchmark.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+df = df0
+df.append(df1)
+df.append(df2)
+df.append(df3)
+
+## Assert location of particles
+assert round(df['coord_x'].min() - 3.285579264425135, 8) == 0.0
+assert round(df['coord_x'].max() - 3.974750713825758, 8) == 0.0
+assert round(df['coord_y'].min() - (-2.204612536254341), 8) == 0.0
+assert round(df['coord_y'].max() - (-1.6069603216032218), 8) == 0.0
```
Comment (jgiven100) [inline explicit_single_phase/validation/sliding_block_inclined_boundary/test_benchmark.py:21]: So these values are **not** based on the kinematic analytic solution, right? Due to the time stepping error, it seems like we shouldn't be using the "real" solution right now.

Comment (bodhinandach) [reply explicit_single_phase/validation/sliding_block_inclined_boundary/test_benchmark.py:21]: This one is fixed after the output stamp is different.

File: explicit_single_phase/validation/sliding_block_inclined_boundary/mpm-file.json (hunk 1 above)
Comment (bodhinandach) [reply explicit_single_phase/validation/sliding_block_inclined_boundary/mpm-file.json:66]: We can run both, the results will be the same. They are just different way to impose velocity constraints

File: explicit_single_phase/validation/uniaxial_stress/results.ipynb [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.5"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```
Comment (jgiven100) [inline explicit_single_phase/validation/uniaxial_stress/results.ipynb]: lets remove this

Comment (jgiven100) [reply explicit_single_phase/validation/uniaxial_stress/results.ipynb]: we will use a `README.md` file to show the outputs

File: explicit_single_phase/validation/uniaxial_traction/nodal-forces-results.ipynb [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.6"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```
Comment (jgiven100) [inline explicit_single_phase/validation/uniaxial_traction/nodal-forces-results.ipynb]: same comment, lets remove and replace with README

File: explicit_single_phase/validation/uniaxial_traction/particle-traction-results.ipynb [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.5"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```
Comment (jgiven100) [inline explicit_single_phase/validation/uniaxial_traction/particle-traction-results.ipynb]: remove

**Review by jgiven100 — APPROVED** [review]

**Comment by bodhinandach** [comment]
Thanks @jgiven100 !

**Pull request merged**